witness/README.md

```diff
@@ -46,3 +73,10 @@ defined above.  Currently it contains implementations in the following languages
   This can be deployed as a smart contract on the Ethereum blockchain.  There are 
   two main contracts, each of which supports one of the consistency checks 
   described above.
+
+## Feeders
+
+* [Go SumDB](https://github.com/google/trillian-examples/tree/master/sumdbaudit/witness) :
+    a custom feeder for sum.golang.org that can be easily deployed with this generic witness implementation
```

Comment:
Last nit: this readme is for multiple witness implementations, so it's not clear here which one you mean. :)

Reply:
Made more specific.